GitCommands/DiffMergeTools/Araxis.cs

```diff
@@ -1,24 +0,0 @@
-using System.Collections.Generic;
```

Comment:
Chose to remove Araxis as it isn't fully configured as diff/merge tool. I've looked at the tool's configuration and it didn't look trivial.
We can add the tool back, if users request it.

Reply:
araxis is supported by Git natively, so .cmd is not needed

Reply:
https://www.araxis.com/merge/documentation-windows/integrating-with-other-applications.en#GitExtensions

The builtin support in master is not up to date
Remove 6.5 references, released 2003-2006

Reply:
I still believe Araxis should be kept, see the configuration in the link.
Cannot be used much though...